### PR TITLE
improve RHOS Ansible docs

### PR DESCRIPTION
**Pull request by jacobdotcosta**

### CODE REVIEW

File: ansible/playbook/openstack/README.adoc [hunk 1]
```diff
@@ -146,11 +146,48 @@ openstack/vm : include_tasks ---------------------------------------------------
 
 === Connect to the new instance
 
-Since all the information related to the host will be managed by our ansible passwordstore roles, which also stores the ssh public and secret keys locally on the `~/.ssh` folder, to login to the newly created VM is as simple as launching the following command.
+All the information related to the host will be managed by our ansible passwordstore roles, which also stores the ssh public and secret keys locally on the `~/.ssh` folder. On the other hand a _bash_ script has been created to make it easier to perform this connection.
```
Comment (cmoulliard) [inline ansible/playbook/openstack/README.adoc:149]: Can you include the link(s) to the Ansible passwordstore roles ?

Comment (cmoulliard) [reply ansible/playbook/openstack/README.adoc:149]: Include the link to the bash script (= passstore_ssh script I suppose)

File: ansible/playbook/openstack/README.adoc [hunk 2]
```diff
@@ -146,11 +146,48 @@ openstack/vm : include_tasks ---------------------------------------------------
...
 
+To SSH connect to a VM use the `tools/passstore-vm-ssh.sh` bas script.
```
Comment (cmoulliard) [inline ansible/playbook/openstack/README.adoc:151]: "bas" -> "bash"

File: ansible/playbook/openstack/README.adoc [hunk 3]
```diff
@@ -146,11 +146,48 @@ openstack/vm : include_tasks ---------------------------------------------------
...
+
+| 1: `VM_PROVIDER`
+
+[.fuchsia]#string# / [.red]#required# 
+a| Cloud provider
+
+Choices: 
+
+* `hetzner`
+* `openstack`
```
Comment (cmoulliard) [inline ansible/playbook/openstack/README.adoc:168]: Can we ssh to the IBM cloud VM ?

Comment (jacobdotcosta) [reply ansible/playbook/openstack/README.adoc:168]: I don't think so, I didn't find a way of doing it.

File: ansible/playbook/openstack/README.adoc (hunk 1 above)
Comment (jacobdotcosta) [reply ansible/playbook/openstack/README.adoc]: Added links to the passwordstore roles and playbooks folder. Also added links to the bash script and it's documentation.

**Pull request merged**